src/Resources/ProductLists/ProductLists.ts

```diff
@@ -0,0 +1,53 @@
+import AbstractResource from '../AbstractResource';
+import { HttpClientConfig, HttpClientResponse } from '../../Interfaces/HttpClient';
+
+export default class ProductLists extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '';
+        this.defaultAccess = 'member';
+    }
+
+    public getAll: (...args: HttpClientConfig) => HttpClientResponse = this.createMethodFromPartialSpec({
+        method: 'GET',
+        path: '/product-lists',
+    });
```

Comment:
Are these all product lists belonging to a user?

If so, could you put those kind of routes under `Resources/User/ProductLists/ProductLists.ts` please?